platform/mbed_wait_api.h

```diff
@@ -53,18 +53,32 @@ extern "C" {
  *  the accuracy of single precision floating point).
  *
  *  @param s number of seconds to wait
+ * 
+ *  @note
+ *    This func always spin to get an exact number of microseconds, potentially 
+ *    impacting power(like can't deep sleep) and multi-thread performance. 
+ *    You can avoided by Thread::wait().
  */
 void wait(float s);
 
 /** Waits a number of milliseconds.
  *
  *  @param ms the whole number of milliseconds to wait
+ * 
+ *  @note
+ *    This func always spin to get an exact number of microseconds, potentially 
```

Comment:
Please change to "This function always spins to get the exact number of microseconds, which potentially affects power (such as preventing deep sleep) and multithread performance. You can avoid it by using Thread::wait()."